package.json

```
@@ -26,6 +26,7 @@
     "got": "11.0.2",
     "jest": "26.6.3",
     "js-yaml": "3.14.0",
+    "lodash": "^4.17.21",
```

Would be nice to fixate :)

```suggestion
    "lodash": "4.17.21",
```

---

updated :)